README.md

```diff
@@ -48,19 +48,19 @@ urlpatterns = [
     ...
     url(r'^api/password_reset/', include('django_rest_passwordreset.urls', namespace='password_reset')),
     ...
-]    
+]
 ```
 **Note**: You can adapt the url to your needs.
 
 ### Endpoints
 
 The following endpoints are provided:
 
- * `POST ${API_URL}/reset_password/` - request a reset password token by using the ``email`` parameter
- * `POST ${API_URL}/reset_password/confirm/` - using a valid ``token``, the users password is set to the provided ``password``
- * `POST ${API_URL}/reset_password/validate_token/` - will return a 200 if a given ``token`` is valid
+ * `POST ${PASSWORD_RESET_URL}/` - request a reset password token by using the ``email`` parameter
+ * `POST ${PASSWORD_RESET_URL}/confirm/` - using a valid ``token``, the users password is set to the provided ``password``
+ * `POST ${PASSWORD_RESET_URL}/validate_token/` - will return a 200 if a given ``token`` is valid
```

Comment:
`${API_URL}/reset_password/` is more explicit and obvious to a user, while hiding these behind a variable would obfuscate it for users and make the documentation harder to understand. I would therefore not apply this change.

Reply:
Thanks for the feedback. I updated the pull request accordingly.